types/chrome/index.d.ts

```diff
@@ -1595,6 +1595,13 @@ declare namespace chrome.declarativeContent {
     /** Declarative event action that shows the extension's page action while the corresponding conditions are met. */
     export class ShowPageAction { }
 
+    /** Declarative event action that changes the icon of the page action while the corresponding conditions are met. */
+    export class SetIcon {
+        constructor (options?: {
```

Comment:
From the document it seems 
`Either an ImageData object or a dictionary {size -> ImageData} representing an icon to be set` so should this be `constructor (options?: ImageData | { [size: string]: ImageData })` instead

Reply:
The dictionary `{size -> ImageData}` is still passed in as a part of the options object:
```
new SetIcon({
imageData: {
'16': iconImageData
}
});
```